test/dependencies.jl

```diff
@@ -69,7 +69,7 @@ end
                 Dependency("LibOSXUnwind_jll")
             ]
             platform = Linux(:i686, libc=:musl)
-            @test_logs (:warn, "Dependency LibOSXUnwind_jll does not have a mapping for LibOSXUnwind!") setup_dependencies(prefix, getpkg.(dependencies), platform)
+            @test_logs (:warn, "Dependency LibOSXUnwind_jll does not have a mapping for artifact LibOSXUnwind!") setup_dependencies(prefix, getpkg.(dependencies), platform)
```

Comment:
The test will need to be adapted, too.
```suggestion
            @test_logs (:warn, "Dependency LibOSXUnwind_jll does not have a mapping for artifact LibOSXUnwind; perhaps it was not built for platform Linux(:i686, libc=:musl)") setup_dependencies(prefix, getpkg.(dependencies), platform)
```